hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/om/TestOmSnapshot.java

```diff
@@ -231,7 +230,7 @@ private void init() throws Exception {
 
     // stop the deletion services so that keys can still be read
     keyManager.stop();
-    preFinalizationChecks();
+//    preFinalizationChecks();
```

Comment:
Please uncomment this.

Reply:
done in 6848b7f36e1397d4c5b71b4009903ca4b6332c93